cla_public/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -4369,6 +4381,159 @@ msgstr "Chwiliwch am gyfryngwr teulu yn eich ardal chi i ganfod a ydych yn gymwy
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Non-accessible content"
+msgstr "Cynnwys nad oes modd cael gafael arno"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "The content listed below is non-accessible for the following reasons."
+msgstr "Nid oes modd cael gafael ar y cynnwys a restrir isod am y rhesymau canlynol."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Non-compliance with the accessibility regulations"
+msgstr "Ddim yn cydymffurfio â’r rheoliadau hygyrchedd"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Users are not able to turn off, adjust or extend the time in which inactivity will result in a loss of data. This fails WCAG 2.1 2.2.1 A (Timing Adjustable)."
+msgstr "Ni all defnyddwyr ddiffodd, addasu nac ymestyn yr amser pan fydd anweithgarwch yn arwain at golli data. Mae hyn yn methu WCAG 2.1 2.2.1 A (Addasu’r Amseriad)."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Some page elements are not available in Welsh. This fails WCAG 2.1 3.1.1 A (Language of Page)."
+msgstr "Nid yw rhai elfennau tudalen ar gael yn Gymraeg. Mae hyn yn methu WCAG 2.1 3.1.1 A (Iaith Tudalen)."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "When a user enters figures into form fields it is not clear that the field does not accept currency symbols. This fails WCAG 2.1 2.4.6 AA (Headings and Labels)."
+msgstr "Pan fydd defnyddiwr yn rhoi ffigurau mewn meysydd ffurflen, nid yw’n glir nad yw’r maes yn derbyn symbolau arian. Mae hyn yn methu WCAG 2.1 2.4.6 AA (Penawdau a Labeli)."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Disproportionate burden"
+msgstr "Baich anghymesur"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Not applicable"
+msgstr "Ddim yn berthnasol"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Content that’s not within the scope of the accessibility regulations"
+msgstr "Cynnwys nad yw o fewn cwmpas y rheoliadau hygyrchedd"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "How we tested this service"
+msgstr "Sut y buom yn profi’r gwasanaeth hwn"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "July"
+msgstr "Gorffennaf"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "September"
+msgstr "Medi"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "December"
+msgstr "Rhagfyr"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "This service was last tested on %(day)s %(month)s %(year)s."
+msgstr "Cafodd y gwasanaeth hwn ei brofi ddiwethaf ar %(day)s %(month)s %(year)s."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "The test was carried out by the Digital Accessibility Centre (<abbr>DAC</abbr>)."
+msgstr "Cynhaliwyd y prawf gan y Ganolfan Hygyrchedd Digidol (<abbr>DAC</abbr>)."
```

Comment:
Line 109 of the accessibility statement
`This service was last tested on 22 July 2020. The test was carried out by the Digital Accessibility Centre (DAC` ) has been replaced with `We are working to fix the time-out issue listed in this statement`

Reply:
This can stay.